migration-db.md

```diff
@@ -0,0 +1,148 @@
+
+**Note:** Elastio can migrate different databases. For more inforrmation on this topic, visit [docs.elastio.com/src/backup-restore/backup-databases.html](https://docs.elastio.com/src/backup-restore/backup-databases.html){:target="_blank"} 
+
+## How to migrate a PostgreSQL database between two environments: An example
+
+We will use a standard web application developed using Python, Flask, and PostgreSQL to demonstrate this concept. 
+
+This step-by-step guide demonstrates the end-to-end database migration pipeline, including prerequisite requirements, installation, and how to use the Elastio Backup and Restore Database feature. 
+
+### Step 1: Requirements
+
+The following prerequisites are necessary before continuing with the process: 
+
+- 2 database instances (production and test)
+- 1 standard instance or environment (production) to deploy the software application to
+
+### Step 2: Installation
+
+- Download this project to your test environment from this [link](https://www.dropbox.com/s/14gdqj78810udln/IndexedSearchPythonDocker.zip?dl=0){:target="_blank"}
+- Install Docker on your test instance
+- Download and install PostgreSQL on both the database instances, if you don't already have PostgreSQL installed
+- Install Elastio on both database instances and the production server 
+
+**Note:** For steps on how to install the Elastio CLI, visit [docs.elastio.com/src/getting-started/install-cli](https://docs.elastio.com/src/getting-started/install-cli.html){:target="_blank"}
+
+### Step 3: Using Elastio 
+
+As per our example, the first thing to do is to create the phonebook database and import the data on the test DB instance. 
+
+**Note:** Because this example describes how to use Elastio's Database Backup and Restore function, the test data has to be set up, even though, operationally, these databases must be kept in sync across all environments. 
+
+1. After successfully installing the PostgreSQL database server, copy the phone_book.sql file to your chosen path on your DB instance. In our case it is: `/home/ubuntu/postgres`
+
+2. To create a database and import the phone_book.sql file, you must follow this step-by-step guide:
+   
+   1. Run the following command to log into the PostgreSQL interactive terminal program: 
+      ```
+      sudo -i -u postgres psql 
+      ```
+      Enter your password if prompted to do so. 
+
+   2. Type in the following SQL statement to create the new phonebook database (and press \<enter>):
+      ```
+      CREATE DATABASE phonebook;
+      ```
+
+   3. Run the following SQL command to import the phone_book.sql file:
+      ```
+      postgres -f <file path>/phone_book.sql phonebook
+      ```
+\#### This command isn't importing the phone_book.sql file - either the command is wrong or the sql file isn't correct. 
```

Comment:
This comment should be removed. 
I was unable to create and import database using commands above, but I do not have a lot of experience in postgres.
I used commands:
sudo -u postgres createdb phonebook
sudo -u postgres psql phonebook  <  <file path>/phone_book.sql